### ISSUE TITLE
Services and Filters folders

### ISSUE THREAD

**Issue by DaftMonk** [DaftMonk/angular-project-draft#1]
We want at least the client/app/routes and client/app/components folders, but theres arguments against having services and filters folders.

It's hard to think of filters and services as components by themselves. They're globally available across the app, and might be used across many different components. And in large apps you may have a very large number of services and filters that would bloat the components folder.

The services and filters folders would be used for services and filters that are used across the app, not just by a specific component. Component folders could contain services and filters specific to that component.

The alternative would be to put services and filters inside nested folders in components that would categorize them. For example: 

  components/apiRepository/user.service.js
  components/apiRepository/item.service.js
  components/nameFilters/name.filter.js
  components/dateFilters/user-age.filter.js
  components/dateFilters/time-posted.filter.js

Thoughts on which method would be preferred?

**Comment by honkskillet** [comment]
@DaftMonk  Is this a repo where you are testing ideas for the new (modular?) generator-angular-fullstack?

**Comment by DaftMonk** [comment]
@honkskillet Yes, that's right. I can't get too into the details right now, but we're trying to test out a new scaffold using es6 modules that will be a base blueprint. Alternative preprocessors, node backend, database, etc, will be add-ons that will either add new blueprints or override existing ones.

This project has some ways to go, so I'll probably be working on a shorter term solution for the fullstack generator until we get further on this.
